e2e_test/batch/catalog/pg_settings.slt.part

```diff
@@ -14,6 +14,7 @@ postmaster checkpoint_frequency
 postmaster enable_tracing
 postmaster max_concurrent_creating_streaming_jobs
 postmaster pause_on_next_bootstrap
+postmaster use_new_object_prefix_strategy
```

Comment:
Is `use_new_object_prefix_strategy` mutable? I think it is immutable because changing it requires manual intervention to migrate data. We should make it `internal` according to https://github.com/risingwavelabs/risingwave/blob/380912047757c25628e82c69e47a7fe8ef7a4c6d/src/frontend/src/catalog/system_catalog/pg_catalog/pg_settings.rs#L38.

Reply:
If it's immutable, how can we change it if we want to make an old cluster split prefix?

Reply:
The mutable here means whether user can change the parameter and whether the change can take effect, not whether we can change it through manual intervention or directly modifying meta store. For example, `data_directory` is immutable, meaning that user cannot change this parameter and even if user can, it will not work. This is the same for `use_new_object_prefix_strategy`

Reply:
Making the `use_new_object_prefix_strategy` mutable will be dangerous because user can techinically change it via ALTER in SQL and the after a restart, the whole cluster will crash.

Reply:
I see.